lib/irc/IrcServer.js

```diff
@@ -68,6 +68,10 @@ IrcServer.prototype.getIdleTimeoutMs = function() {
     return this.config.ircClients.idleTimeout;
 };
 
+IrcServer.prototype.getReconnectInterval = function() {
```

Comment:
Just so you know, I renamed that - https://github.com/matrix-org/matrix-appservice-irc/issues/134 
It was in seconds in the code and also, the code treated it as seconds.